### PR TITLE
Editorial: add missing constructor property of the global object

### PR DESCRIPTION
**Pull request by aduh95**
`AggregateError` is not listed in section 19.3, while it is specd as available on the global object in section 20.5.7.1.

### CODE REVIEW

**Review by michaelficarra — APPROVED** [review]

**Review by mathiasbynens — APPROVED** [review]

**Comment by aduh95** [comment]
Is there more I can do to have this ready to land?

**Comment by michaelficarra** [comment]
@aduh95 No, it looks good to me. It is just waiting on reviews from the other @tc39/ecma262-editors.

**Review by bakkot — APPROVED** [review]
LGTM, sorry for the delay.

**Comment by bakkot** [comment]
@ljharb I'm assuming the "editor call" label was just for a review ping; I've reviewed it now so I took the label off, but please put it back on if there was more to discuss.

**Pull request merged**
